### PR TITLE
Throw on skip

### PR DESCRIPTION
**Pull request by Jens-G**
Some helpful tips for a successful Apache Thrift PR:

* Did you test your changes locally or using CI in your fork?
* Is the Apache Jira THRIFT ticket identifier in the PR title?
* Is the Apache Jira THRIFT ticket identifier in the commit message?
* Did you squash your changes to a single commit?
* Are these changes backwards compatible? (please say so in PR description)
* Do you need to update the language-specific README?

Example ideal pull request title:

        THRIFT-9999: an example pull request title

Example ideal commit message:

        THRIFT-9999: [summary of fix, one line if possible]
        Client: [language(s) affected, comma separated, use lib/ directory names please]

For more information about committing, see CONTRIBUTING.md

### CODE REVIEW

**Comment by jeking3** [comment]
Cool thanks, I just updated my master so it doesn't say it is a conflict, and I am going to push up c_glib changes shortly, then we'll see where the collisions are.

File: lib/nodejs/lib/thrift/binary_protocol.js [hunk 1]
```diff
@@ -302,8 +302,6 @@ TBinaryProtocol.prototype.getTransport = function() {
 
 TBinaryProtocol.prototype.skip = function(type) {
```
Comment (jeking3) [inline lib/nodejs/lib/thrift/binary_protocol.js:303]: I wonder why this is replicated in more than one place (skip)...

**Comment by jeking3** [comment]
Running this locally, will push anything I can to fix and move forward.

**Comment by jeking3** [comment]
The dlang code needed a fix, working on it..

File: lib/nodejs/lib/thrift/binary_protocol.js (hunk 1 above)
Comment (Jens-G) [reply lib/nodejs/lib/thrift/binary_protocol.js:303]: You're not alone :-)

**Pull request merged**